server/e2e/gql_asset_test.go

```diff
@@ -0,0 +1,236 @@
+package e2e
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/gavv/httpexpect/v2"
+	"github.com/reearth/reearth-cms/server/internal/adapter/gql/gqlmodel"
+	"github.com/reearth/reearth-cms/server/internal/app"
+	"github.com/reearth/reearth-cms/server/pkg/id"
+	"github.com/reearth/reearthx/account/accountdomain"
+	"github.com/stretchr/testify/assert"
+)
+
+// Reference to the user ID defined in gql_workspace_test.go
+var _ = accountdomain.UserID{}
+
+func TestSearchAsset(t *testing.T) {
```

Comment:
create 2 assets with 2 diffrent contet type and make sure the filter return the correct asset